frontend/src/components/core/Input.tsx

```diff
@@ -1,9 +1,14 @@
 import styled from 'styled-components';
 
-const Input = styled.input`
+type Dimensions = {
+  width?: string,
+  height?: string,
+};
+
+const Input = styled.input<Dimensions>`
   box-sizing: border-box;
   border-radius: 0.25rem;
-  border: 3px solid ${({ theme }) => theme.colors.blue};
+  border: 2px solid ${({ theme }) => theme.colors.blue};
```

Comment:
Hm yeah that might be useful at some point (or even just adding the entire border definition into theme - along the same lines we could also add the box-shadow to theme as well since ik that one is pretty heavily copied into various places